test/test1.js

```diff
@@ -0,0 +1,307 @@
+var assert = require('assert');
+var request = require('request');
+
+describe('Personas Module', function() {
+
+  var apos;
+  const basePath = "http://localhost:3000";
+  const userAgents = {
+    desktop: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36",
+    googlebot: "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)",
+    googleImageBot: "Googlebot-Image/1.0"
+  };
+
+  this.timeout(3000);
+
+  after(function() {
+    apos.db.dropDatabase();
+  });
+
+  /// ///
+  // EXISTENCE
+  /// ///
+  it('should be a property of the apos object', function(done) {
+    apos = require('apostrophe')({
+      testModule: true,
+      baseUrl: 'http://localhost:3000',
+
+      modules: {
+        'apostrophe-pages': {
+          park: [{slug: '/', type: 'home'}],
+          types: [
+            {
+              name: 'home',
+              label: 'Home'
+            },
+            {
+              name: 'testPage',
+              label: 'Test Page'
+            }
+          ]
+        },
+        'apostrophe-personas': {
+          personas: [
+            {
+              name: 'employee',
+              label: 'Employee',
+              prefix: '/employee'
+            },
+            {
+              name: 'employer',
+              label: 'Employer',
+              prefix: '/employer'
+            }
+          ]
+        }
+      },
+
+      afterInit: function(callback) {
+        assert(apos.modules['apostrophe-personas']);
+        // Should NOT have an alias!
+        assert(!apos.personas);
+        return callback(null);
+      },
+      afterListen: function(err) {
+        assert(!err);
+        done();
+      }
+    });
+  });
+
+  it('Should load home page', function (done) {
+    return apos.pages.find(apos.tasks.getReq({ locale: 'default-draft' }), { slug: '/' }).toObject(function(err, home) {
+      assert(!err);
+      assert(home.slug === '/');
+      assert(home.type === 'home');
+      done();
+    });
+  });
+
+  it('Test request for persona via http', function (done) {
+    let j = request.jar();
+
+    return request({url: `${basePath}/?persona=employee`, jar: j}, 'GET')
+      .on('response', res => {
+        assert(true);
+        done();
+      })
+      .on('error', err => {
+        console.log("ERR", err);
+        assert(false);
+        done();
```

Comment:
We will never reach done, by your intent, so remove.

Reply:
(I think you kept these because I didn't mention an assert failure will successfully continue the async testing process on its own. It will do that.)